connectivity/netsocket/source/TLSSocketWrapper.cpp

```diff
@@ -41,6 +45,16 @@ TLSSocketWrapper::TLSSocketWrapper(Socket *transport, const char *hostname, cont
     _clicert_allocated(false),
     _ssl_conf_allocated(false)
 {
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+    // It is safe to call psa_crypto_init() any number of times as
+    // defined by the PSA Crypto API. There is no standard "deinit"
+    // function.
+    psa_status_t status = psa_crypto_init();
+    if (status != PSA_SUCCESS) {
+        tr_err("psa_crypto_init() failed (" PRIu32 ")", status);
```

Comment:
Should we use `print_mbedtls_error()` here like we do in other functions?

Reply:
PSA status is different from Mbed TLS error, and `print_mbedtls_error()` uses `mbedtls_strerror()` which doesn't support translating PSA errors.

Reply:
OK, too bad it doesn't support PSA errors.